File: stripe/src/test/java/com/stripe/android/view/ShippingInfoTestActivity.java
package com.stripe.android.view;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.LinearLayout;

import com.stripe.android.R;

/**
 * Activity used to test UI components
 */
public class ShippingInfoTestActivity extends AppCompatActivity {

    private ShippingInfoWidget mShippingInfoWidget;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTheme(R.style.StripeDefaultTheme);
        mShippingInfoWidget = new ShippingInfoWidget(this);
        LinearLayout linearLayout = new LinearLayout(this);
        linearLayout.addView(mShippingInfoWidget);
        setContentView(linearLayout);
    }

    public ShippingInfoWidget getShippingInfoWidget() {
        return mShippingInfoWidget;
    }
}

File: payments-core/src/test/java/com/stripe/android/view/ShippingInfoTestActivity.kt
package com.stripe.android.view

import android.os.Bundle
import android.widget.LinearLayout
import androidx.appcompat.app.AppCompatActivity
import com.stripe.android.R

/**
 * Activity used to test UI components
 */
class ShippingInfoTestActivity : AppCompatActivity() {

    lateinit var shippingInfoWidget: ShippingInfoWidget
        private set

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setTheme(R.style.StripeDefaultTheme)
        shippingInfoWidget = ShippingInfoWidget(this)

        val linearLayout = LinearLayout(this)
        linearLayout.addView(shippingInfoWidget)
        setContentView(linearLayout)
    }
}
